src/components/developers/WebhookPortal.tsx

```
@@ -0,0 +1,443 @@
+                      </button>
+                    </div>
+                  </div>
+
+                  {/* Secret Key Bar */}
+                  <div className="p-2 bg-white border border-gray-300 rounded flex items-center justify-between text-[11px]">
+                    <span className="font-bold text-gray-700 flex items-center gap-1">
+                      <Key className="w-3.5 h-3.5 text-amber-600" />
+                      Signing Secret:
+                    </span>
+                    <span className="font-mono text-gray-800">
+                      {visibleSecretId === ep.id ? ep.secret_key : `${ep.secret_key.slice(0, 10)}••••••••••••`}
```

ESLint

[error] Replace `·?·ep.secret_key` with `⏎························?·ep.secret_key⏎·······················`

(prettier/prettier)